Makefile.am

```diff
@@ -124,9 +124,13 @@ test_basic_tls:
 	fi
 endif
 
+
 test:	memcached-debug sizes testapp
 	$(builddir)/sizes
 	$(builddir)/testapp
+if ENABLE_JEMALLOC
+	@JEMALLOC_TEST=1 prove $(srcdir)/t/jemalloc.t
```

Comment:
Is there another runtime/startitme way to detect if jemalloc exists? This causes this one test to run not in parallel. I spent a lot of time running these tests :)

Reply:
We need to somehow wire the fact jemalloc is linked from configuration to tests. Though a bit of an overkill, a different make target would work I believe?

Reply:
https://github.com/jemalloc/jemalloc/issues/388 - I see a couple decent approaches here. Should be easy to add a `supports_jemalloc()` test to the ones in t/lib/MemcachedTest.pm

Reply:
Looking at the test again.. trying to figure out what the goal of it is. If jemalloc is enabled, it's checking that it's enabled via ldd?

So the supports() test could be looking for something that configure would dump (like a "HAVE_JEMALLOC" in config.h, would be pretty easy to add)

Reply:
maybe ignore this comment and see the other one.